app/src/hooks/useUserBullets.ts

```diff
@@ -17,7 +17,9 @@ export type NoteBullet = {
     images: string[],
     bulletTextContent: string
     bulletLongitude?: number,
-    bulletLatitude?: number
+    bulletLatitude?: number,
+    consecutiveDays?: number,
```

Comment:
Why do we need two attributes on every single notes for streaks? @a4 I definitely need a logic explanation for why we chose to do it this way. 

Can't we store consecutiveDays solely in the userProfile for streaks?